src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -251,7 +262,7 @@ trait ClusterComponent extends LeoComponent {
     /* WARNING: The init bucket and SA key ID is secret to Leo, which means we don't unmarshal it.
      * This function should only be called at cluster creation time, when the init bucket doesn't exist.
      */
-    private def marshalCluster(cluster: Cluster, initBucket: String, serviceAccountKeyId: Option[ServiceAccountKeyId]): ClusterRecord = {
+    private def marshalCluster(cluster: Cluster, initBucket: String, serviceAccountKeyId: Option[ServiceAccountKeyId], googleClientId: Option[GoogleClientId]): ClusterRecord = {
```

Comment:
I wasn't sure if the `googleClientId` should be exposed in the `Cluster` object or not. I tend to think it's kind of secret, so we persist in the DB but don't unmarshal it back.